integration_tests/src/main/python/arithmetic_ops_test.py

```diff
@@ -824,6 +824,14 @@ def test_bit_not(data_gen):
     assert_gpu_and_cpu_are_equal_collect(
             lambda spark : unary_op_df(spark, data_gen).selectExpr('~a'))
 
+
+@pytest.mark.skipif(is_before_spark_350(),
```

Comment:
The spark code says that this has been around since 3.0.0? So how does this compile if we are not restricting the compilation to just 3.5.0+